wallet/quickstart/javascript.md

```diff
@@ -0,0 +1,130 @@
+---
+description: Integrate your JavaScript dapp with MetaMask.
+sidebar_position: 1
+---
+
+# JavaScript quickstart
+
+This page provides code samples to quickly connect to your users' MetaMask accounts from a
+JavaScript dapp.
+You can:
+
+- [Connect to the MetaMask browser extension](#connect-to-the-metamask-browser-extension) directly
+  using the MetaMask APIs.
+- [Connect to the MetaMask extension and MetaMask Mobile](#connect-to-the-metamask-extension-and-metamask-mobile)
+  using the SDK.
+
+## Connect to the MetaMask browser extension
+
+Connect to the MetaMask browser extension directly using the
+[MetaMask Ethereum provider API](../concepts/apis.md).
+You must [detect MetaMask](../how-to/detect-metamask.md) in the user's browser, then
+[access the user's accounts](../how-to/access-accounts.md) by prompting them to connect to MetaMask.
+
+First, install the [`@metamask/detect-provider`](https://github.com/MetaMask/detect-provider)
```

Comment:
Even with only two steps, it might be better to number these statements to make the content more identifiable and accessible.